operator/roles/ansible-service-broker/tasks/main.yml

```diff
@@ -5,6 +5,31 @@
   set_fact:
     api_groups: "{{ lookup('k8s', cluster_info='api_groups') }}"
 
+# reconciled_generation - the last reconciled spec generation of a given CR
+# generation - the current generation of a given CR
+# These values are useful because if the current generation value is different
+# than the reconciled_generation value, we know that the CR's spec has been
+# changed, requiring a bounce of the broker's pod to accept new configuration.
+- name: Set reconciled_generation and generation facts
+  set_fact:
+    generation: "{{
+      lookup('k8s',
+      api_version='osb.openshift.io/v1alpha1',
+      kind='AutomationBroker',
+      namespace=broker_namespace,
+      resource_name=broker_name).metadata.generation
+    }}"
+    reconciled_generation: "{{
+      lookup('k8s',
+      api_version='osb.openshift.io/v1alpha1',
+      kind='AutomationBroker',
+      namespace=broker_namespace,
+      resource_name=broker_name).status.reconciledGeneration | default('')
+    }}"
+
+- set_fact:
```

Comment:
You will need a name on this to pass any linting of this we do (this will be important when we tie this operator into openshift ci).